cdap-docs/hydrator-manual/build.sh

```diff
@@ -233,7 +233,7 @@ function download_includes() {
   download_md_file core-plugins File-batchsource.md
   download_md_file core-plugins FTP-batchsource.md
   download_md_file core-plugins GroupByAggregate-batchaggregator.md
-  download_md_file core-plugins HDFSFileMoveAction-action.md
+  download_md_file core-plugins HDFSMove-action.md
```

Comment:
That's in the other PR. This is just to get a clean build in the main release line.